apps/vessel-history/features/dataviews/dataviews.config.ts

```diff
@@ -54,11 +54,11 @@ export const OFFLINE_LAYERS: GeneratorConfig[] = [
 export const DEFAULT_BASEMAP_DATAVIEW_ID = WORKSPACE_ENV === 'development' ? 90 : 173
 // export const DEFAULT_VESSEL_DATAVIEW_ID = WORKSPACE_ENV === 'development' ? 291 : 268 // Vessel with port visits c3 and c4
 export const DEFAULT_VESSEL_DATAVIEWS = {
-  'port-inspector': WORKSPACE_ENV === 'development' ? 176 : 268, // Vessel with port visits c2
+  'port-inspector': WORKSPACE_ENV === 'development' ? 291 : 268, // Vessel with port visits c2
```

Comment:
port inspectors should still use the dataviews with port visits c2 (176 and 268)

Reply:
yes